src/attributes/attributes_cmd.c

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file
```

Comment:
Can you pull the doxygen tags from the header? I'd like to stick with the prescribed header on the samples.

Reply:
Resolved in https://github.com/aws-samples/aws-cloudhsm-pkcs11-examples/commit/4d4db3806c88d6330cb726997c1a193b74934e5f